dataland-frontend/tests/e2e/specs/lksg/DataIntegrity.ts

```diff
@@ -40,6 +62,7 @@ describeIf(
           return uploadCompanyViaApi(token, generateDummyCompanyInformation(testCompanyName));
         })
         .then((storedCompany) => {
+          cy.ensureLoggedIn(admin_name, admin_pw);
```

Comment:
Just change the `ensureLoggedIn` in the `beforeEach` hook.  Then you don't need this here.

Reply:
beforeEach() already has an `ensureLoggedIn()` code. This line (65) I believe is intentional.

```
beforeEach(() => {
      cy.ensureLoggedIn(uploader_name, uploader_pw);
});
```

Reply:
Yes and I suggest that we log in as "admin" in the `beforeEach` hook.   Then we don't need to re-login as admin here and can delete the statement.

Reply:
Ah. Gotcha.

Reply:
👍